part-6/r-introduction.md

```diff
@@ -1,15 +1,1653 @@
 # R - Introduction
 
-## R Toolbox for MoBi®‌‌‌
```

Comment:
New R-docu, which is in fact just all "ospsuite-R" vignettes put together (with minimal adjustments)